pyphare/pyphare/pharein/initialize/__init__.py

```
@@ -1,3 +1,76 @@
+import pybindlibs.dictator as pp
 from . import general, hybrid, mhd
 
-__all__ = ["general", "hybrid", "mhd"]
+__all__ = ["general", "hybrid", "mhd", "DictPopulator"]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)